packs/other-effects/effect-aquatic-combat.json

```diff
@@ -103,6 +103,50 @@
                     }
                 ],
                 "uuid": "Compendium.pf2e.conditionitems.Item.Off-Guard"
+            },
+            {
+                "key": "DamageAlteration",
+                "mode": "override",
+                "predicate": [
+                    {
+                        "or": [
+                            "item:trait:fire",
```

Comment:
We shouldn't do the fire trait, just fire damage. A strike with the fire trait still has its non-fire effects.